Source/WebCore/platform/graphics/MediaSourcePrivate.h

```diff
@@ -46,14 +46,16 @@ class LegacyCDMSession;
 #endif
 
 class WEBCORE_EXPORT MediaSourcePrivate
-    : public RefCounted<MediaSourcePrivate>
-    , public CanMakeWeakPtr<MediaSourcePrivate> {
+    : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<MediaSourcePrivate> {
```

Comment:
With this change, we no longer know where will be destroyed this private.
Looking at the members, this seems fine.
Except, if the destructor is doing something things that need to be done in a specific thread.

Reply:
The privates should never contain any thread-specific objects (nor DOM objects) as they are all in platforms.